demos/minimap/minimap.js

```diff
@@ -0,0 +1,247 @@
+/**
+* Blockly Demos: Code
+*
+* Copyright 2017 Google Inc.
+* https://developers.google.com/blockly/
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* @fileoverview JavaScript for Blockly's Minimap demo.
+* @author karnpurohit@gmail.com (Karan Purohit)
+*/
+'use strict';
+
+/**
+* Creating a seperate namespace for minimap.
+*/
+var Minimap = {};
+
+/**
+* initilize the workspace and minimap.
+* @param {Workspace} workspace The main workspace of the user
```

Comment:
Basic style changes, to apply broadly:
- End comments with periods, even for params.
- Capitalize wors at the beginning of comments.
- Spaces on both sides of operators (=, -, /, etc.)
- Spaces after commas between function arguments.